frontend/src/apiMocks/mockData.ts

```diff
@@ -0,0 +1,148 @@
+import { CustomAliasData, RandomAliasData } from "../hooks/api/aliases";
+import { ProfileData } from "../hooks/api/profile";
+
+export const mockIds = ["empty", "onboarding", "some", "full"] as const;
+
+export const profiles: Record<typeof mockIds[number], ProfileData> = {
+  empty: {
+    api_token: "empty",
+    avatar: "https://profile.accounts.firefox.com/v1/avatar/e",
+    bounce_status: [false, ""],
+    date_subscribed: null,
+    has_premium: false,
+    id: 0,
+    next_email_try: "2020-04-09T00:00:00.000Z",
+    onboarding_state: 0,
+    server_storage: true,
+    subdomain: null,
+  },
+  onboarding: {
+    api_token: "onboarding",
+    avatar: "https://profile.accounts.firefox.com/v1/avatar/o",
+    bounce_status: [false, ""],
+    date_subscribed: "2020-04-09T00:00:00.000Z",
+    has_premium: true,
+    id: 0,
+    next_email_try: "2020-04-09T00:00:00.000Z",
+    onboarding_state: 0,
+    server_storage: true,
+    subdomain: null,
+  },
+  some: {
+    api_token: "some",
+    avatar: "https://profile.accounts.firefox.com/v1/avatar/s",
+    bounce_status: [false, ""],
+    date_subscribed: "2020-04-09T00:00:00.000Z",
+    has_premium: true,
+    id: 0,
+    next_email_try: "2020-04-09T00:00:00.000Z",
+    onboarding_state: 3,
+    server_storage: true,
+    subdomain: null,
+  },
+  full: {
+    api_token: "full",
+    avatar: "https://profile.accounts.firefox.com/v1/avatar/f",
+    bounce_status: [true, "soft"],
+    date_subscribed: "2020-04-09T00:00:00.000Z",
+    has_premium: true,
+    id: 0,
+    next_email_try: "2020-04-09T00:00:00.000Z",
+    onboarding_state: 3,
+    server_storage: true,
+    subdomain: "mydomain",
+  },
+};
```

Comment:
6. This is an example of the default mock data for the `/api/v1/profiles/` endpoint. Every key is a different mock ID, and it's typed, so if the API response changes, TypeScript will warn us about updating these mocks. Below this is the data for the other endpoints. As you can see, the user identified by the mockId `empty` has no random aliases, neither does the one identified by `onboarding`, whereas the user identified by `some` has two.